laterpay/application/Compatibility/Migrate.php

```diff
@@ -312,30 +303,23 @@ public function migrate_terms_table( $offset ) {
     /**
      *  Drop laterpay custom tables.
      *
-     * @wp-hook wp_ajax_laterpay_drop_custom_tables
-     * @param LaterPay_Core_Event $event
-     *
-     * @return void
+     * @return bool
      */
-    public function drop_custom_tables( LaterPay_Core_Event $event ) {
+    public function drop_custom_tables() {
 
-        if ( ! is_user_logged_in() || ! current_user_can( 'manage_options' ) ) {
-            return;
-        }
+        if ( laterpay_is_migration_complete() ) {
```

Comment:
Should continue checking `current_user_can( 'manage_options' )`?

Reply:
`drop_custom_tables` won't be called if the user doesn't have permission.

https://github.com/laterpay/laterpay-wordpress-plugin/pull/1050/files#diff-1de5cf8560a526c845948882499a3f25R94

Won't that be enough?

Reply:
If in installation script we are checking permission then it's fine.

Reply:
The ajax request ( which will call `drop_custom_tables ` if migration is complete ) won't process further if a user doesn't have permission. So no issue.